Dockerfile

```diff
@@ -15,6 +15,7 @@ WORKDIR /go/src/github.com/eclipse/che-machine-exec/
 COPY . .
 RUN CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-w -s' -a -installsuffix cgo -o che-machine-exec .
 
-FROM registry.centos.org/centos:7
+FROM alpine:3.8
```

Comment:
curious, we're not able to use scratch image ?
https://medium.com/@chemidy/create-the-smallest-and-secured-golang-docker-image-based-on-scratch-4752223b7324#ae0c

Reply:
Yes, thanks, We can. Size will be really less: from 30.3 Mb to 29.9 Mb.

Reply:
che-machine-exec binary -> 24Mb

Reply:
We use client libraries to work with kubernetes and docker. That's why I think it has such size.